syntax = "proto3";
package prototypes;

message Bench {
  bytes blob = 1;
  uint64 creation_timestamp_ns = 2; // nanoseconds since UNIX epoch
  uint32 total_number_of_messages = 3;
  uint32 message_number = 4;
}
